include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -103,6 +125,24 @@ struct CXXMethodBridging {
     // The first character is always lowercase.
     output.front() = std::tolower(output.front());
 
+    if (classifyNameKind() == NameKind::snake) {
+      for (std::size_t i = 0; i < output.size(); i++) {
+        size_t next = i + 1;
+        if (output[i] == '_') {
+          // if the first element is an underscore, remove it
+          if (i == 0) {
+            output.erase(i, 1);
+          } else {
+            // if the current element is an underscore, capitalize the element
+            // next to it, and remove the extra element
+            output[i] = std::toupper(output[next]);
```

Comment:
Will this crash if we have a trailing underscore? For example `get_x_()`?

Reply:
When I tested I didnt end up getting a crash but i will verify

Reply:
add a test for trailing snake case similar to the other getX and setX tests